### PR TITLE
Fix Bugs on editing

### PR DESCRIPTION
**Pull request by Arthurlpgc**
Fixes 2 bugs, one is: #238 and the other is the fact that strings are not being translated for field names

### CODE REVIEW

File: vitaes-webapp/src/Builder/CvItemForm/index.js [hunk 1]
```diff
@@ -54,10 +53,9 @@ class CvItemForm extends Component {
 
   getEventExpander(index) {
     return () => {
-      if (this.props.label === this.props.chosenLabel) {
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/Builder/CvItemForm/index.js]: This wasn't reliable behavior, so modifying

File: vitaes-webapp/src/Builder/CvItemForm/index.js [hunk 2]
```diff
@@ -93,11 +91,7 @@ class CvItemForm extends Component {
     const { toAdd } = this.state;
     for (const item of this.props.fields) {
       if (toAdd[item[0]] === undefined) {
-        if (item[0] === 'name') {
-          toast.error(`${translate('mandatory_field')}: ${translate('title')}`);
-        } else {
-          toast.error(`${translate('mandatory_field')}: ${capitalize(item[0])}`);
-        }
+        toast.error(`${translate('mandatory_field')}: ${item[2]}`);
```
Comment (rafaellsma) [inline vitaes-webapp/src/Builder/CvItemForm/index.js]: why index 2? Is there index 2 always on array?

File: vitaes-webapp/src/Builder/CvItemForm/index.js [hunk 3]
```diff
@@ -54,10 +53,9 @@ class CvItemForm extends Component {
 
   getEventExpander(index) {
     return () => {
-      if (this.props.label === this.props.chosenLabel) {
-        if (!this.addField()) {
```
Comment (rafaellsma) [inline vitaes-webapp/src/Builder/CvItemForm/index.js]: This method is used in another place?

Comment (Arthurlpgc) [reply vitaes-webapp/src/Builder/CvItemForm/index.js]: Yes on line 215

File: vitaes-webapp/src/Builder/CvItemForm/index.js (hunk 2 above)
Comment (Arthurlpgc) [reply vitaes-webapp/src/Builder/CvItemForm/index.js]: it is the position that name is on, it is the tech debt discussed on the meeting today

**Pull request merged**